modules/layers/src/bitmap-layer/bitmap-layer.js

```diff
@@ -108,6 +108,22 @@ export default class BitmapLayer extends Layer {
     }
   }
 
+  getPickingInfo({info}) {
+    // Calculate uv and pixel in bitmap
+    const [u, v] = info.color;
+    info.bitmapUV = [u / 256, v / 256];
+
+    const uniforms = this.state && this.state.model && this.state.model.uniforms;
+    const {bitmapTexture} = uniforms || {};
+    if (bitmapTexture) {
+      info.bitmapWidth = bitmapTexture.width;
+      info.bitmapHeight = bitmapTexture.height;
+      info.bitmapPixel = [(u / 256) * bitmapTexture.width, (v / 256) * bitmapTexture.height];
```

Comment:
Shall we read the pixel color from the image while we are at it?

Reply:
Yes I also thought about this. Maybe you have better ideas, but I didn't see a good way.

Reading back from the texture is an expensive operation, so something we want to avoid in case the data is not needed. I suppose we could control it with a bitmap layer prop.

Also details of read back can depend on the format of the texture - if data on GPU is stored bytes, floats, high-bit integers etc.

Reply:
You can add a getter like some of the MVTLayer callbacks:

```js
Object.defineProperty(bitmap, 'pixelColor', {
  get: () => readPixel(...)
});
```

Reply:
- While that could work, isn't it best if the pick info object is simple JS object that can be copied and worked naturally without hidden functionality that triggers expensive GPU operations?
- Also I expect that this callback would still have to bind the texture and would still hold on to it?


I think it could be cleaner to offer a `pickPixelColor` prop that performs this operation and included `pixelColor` in the pickInfo object.

Alternatively:
- If the BitmapLayer only has one bitmap (no instances) maybe offer a function or accessor on the BitmapLayer instead to get the current bitmap?
- Maybe then we can just add a mention of luma `readPixel` function in the BitmapLayer docs.

Reply:
> maybe offer a function or accessor on the BitmapLayer instead to get the current bitmap?

`layer.props.image`